Remotion/ObjectBinding/Web/Services/IBocAutoCompleteReferenceValueWebService.cs

```diff
@@ -29,13 +29,16 @@ public interface IBocAutoCompleteReferenceValueWebService : IBocReferenceValueBa
     /// Retrieves a list of objects based on the <paramref name="searchString"/> and the search context (i.e. the <paramref name="businessObjectClass"/> etc).
     /// </summary>
     /// <param name="searchString">The <see cref="string"/> all returned values must match.</param>
+    /// <param name="completionSetOffset">
+    ///   The number of items to skip before returning results or <see langword="null" /> if no offset should be used and results should be returned from the beginning.
+    /// </param>
```

Comment:
```suggestion
    /// <param name="completionSetOffset">
    ///   The number of items to skip before returning results or zero if no offset should be used and results should be returned from the beginning.
    /// </param>
```
it's a non-nullable int.